docs/layout/spacing/README.md

```diff
@@ -0,0 +1,205 @@
+| pr            | padding-right                | cdr-pr-space-one-x            |
+| pb            | padding-bottom               | cdr-pb-space-one-x            |
+| pl            | padding-left                 | cdr-pl-space-one-x            |
+| px            | padding-left & padding-right | cdr-px-space-one-x            |
+| py            | padding-top & padding-bottom | cdr-py-space-one-x            |
+| mt            | margin-top                   | cdr-mt-space-one-x            |
+| mr            | margin-right                 | cdr-mr-space-one-x            |
+| mb            | margin-bottom                | cdr-mb-space-one-x            |
+| ml            | margin-left                  | cdr-ml-space-one-x            |
+| mx            | margin-left & margin-right   | cdr-mx-space-one-x            |
+| my            | margin-top & margin-bottom   | cdr-my-space-one-x            |
+| inset         | padding                      | cdr-space-inset-one-x         |
+| inset-squish  | padding                      | cdr-space-inset-squish-one-x  |
+| inset-stretch | padding                      | cdr-space-inset-stretch-one-x |
+
+For a full list of space utility classes, see the [CSS Utility Class](../../components/utilities/) document.
+
+### Breakpoint (Optional)
+
+Sometimes your content may need to be spaced differently at specific breakpoints. To accommodate such different spacing needs, spacing utilities are available at each of our defined breakpoints by concatenating the ‘@’ symbol with the desired breakpoint to the end of the class name.
+
+-  `xs` - applies the spacing **for the extra small breakpoint ( < 768px)**
+-  `sm` - applies the spacing **for the small breakpoint (768px <= X < 992px)** 
+-  `md` - applies the spacing **for the medium breakpoint (992px <= X < 1232px)**
+-  `lg `- applies the spacing **for the large breakpoint ( > 1232px)**
+
+For example, `.cdr-space-inset-eighth-x@md` will apply if the user’s screen is currently at the medium breakpoint (within 992px and 1232px wide).
+
+For more information on breakpoints and responsive design, see the [Responsive Layout](../responsive) article.
+
+To add right margin to the Cedar button component at the medium breakpoint and up, but bottom margin to at the small breakpoint and below:
+
+
+For a full list of space utility classes, see the [CSS Utility Class](../../components/utilities/) document.
+
+<cdr-doc-example-code-pair :background-toggle="false" :sandbox-data="$page.frontmatter.sandboxData" >
+
+```vue
+  <div>
+    <cdr-button 
+     full-width="@xs"
+      space="
+        cdr-mb-space-one-x@xs
+        cdr-mb-space-one-x@sm
+        cdr-mr-space-one-x@sm
+        cdr-mr-space-one-x@md
+        cdr-mr-space-one-x@lg
+      "
+    > 
+    A sample Cedar button
+    </cdr-button>
+    <cdr-button 
+    modifier="secondary"
+    full-width="@xs"
+    > 
+    Another sample Cedar button
+    </cdr-button>
+  </div>
+
```

Comment:
I don't see the sample Cedar button screenshot in the doc.

Reply:
there is a code block at the bottom --

```
<cdr-button 
  space="
    cdr-mb-space-one-x@xs
    cdr-mb-space-one-x@sm
    cdr-mr-space-one-x@md
    cdr-mr-space-one-x@lg
  "
> 
 Your buttons text
</cdr-button>
``` 

I replaced this with a sandbox that actually demos what is going on